.install/macos.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 export HOMEBREW_NO_AUTO_UPDATE=1
 GNUBIN=$(brew --prefix)/opt/make/libexec/gnubin
+LLVM=/usr/local/opt/llvm@16/bin
```

Comment:
I believe `/usr/local` is what `brew --prefix` returns. Let's put it into a variable and use it for both paths

Reply:
i tried `$(brew --prefix llvm@16)/bin/clang` as mentioned [here](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#installed-software) and it didn't work.
i'll try it without the `llvm@16`

Reply:
Make sense it wouldn’t work before you install llvm. You can also try to move these lines to after the installation. I didn’t know this command can be package-specific